inductiva/resources/machines.py

```diff
@@ -0,0 +1,107 @@
+"""Containes Machines class to manage resources."""
+
+from absl import logging
+from typing import Optional
+from uuid import UUID
+import uuid
+import inductiva
+from inductiva import api
+
+from inductiva.client import ApiClient, ApiException
+from inductiva.client.apis.tags.instance_api import InstanceApi
+from inductiva.client.model.instance import Instance
+from inductiva.client.model.instance_group import InstanceGroup
+
+
+class Machines():
+    """Class to Google Cloud manage resources."""
+
+    def __init__(
+        self,
+        machine_type: str,
+        executer: str,
+        num_machines: int = 1,
+        spot: bool = False,
+        disk_size_gb: int = 30,
+        label: str = None,
+        resource_pool_id: Optional[UUID] = None,
```

Comment:
If this will be the id of this set of MachineGroup, then let's call this, machine_group_id, otherwise we are using two different names (machine_group and resource_pool, for the same thing).